design-system/materials/custom-properties.css

```diff
@@ -151,7 +180,11 @@
   --background-color-for-button-as-primary-when-active: #15a390;
   --background-color-for-button-as-primary-when-hovered: #17ab97;
   --background-color-for-button-as-icon-as-primary-when-active: #15a390;
-  --background-color-for-button-as-icon-as-primary-when-hovered: #00b39e;
+  --background-color-for-button-as-icon-as-primary-when-hovered: hsl(
```

Comment:
@commercetools/shield-team-fe Out of curiosity, is there a reason that these named values are setting `hsl` values directly, when they are meant to be the same as the "base" variable, in this case `var(--color-primary)`?

As far as theming is concerned, I would think that referencing the var/custom-property would be more maintainable than declaring the `hsl` in multiple places, and would better enforce consistency in using the correct color values

Reply:
I now realize that the answer to this is that these values are defined consistently in `internals/definition.yaml`, which is the actual source of truth for what's in these files, which makes a whole lot of sense